cmd/sdkch/main.go

```diff
@@ -0,0 +1,277 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+const (
+	entriesDirName = ".pending"
+)
+
+var (
+	progName string
+
+	entriesDir string
+
+	// sections name-title map
+	sections = map[string]string{
+		"breaking":     "Breaking Changes",
+		"features":     "New features",
+		"improvements": "Improvements",
+		"bugfixes":     "Bugfixes",
+	}
+	// stanzas name-title map
+	stanzas = map[string]string{
+		"gaia":       "Gaia",
+		"gaiacli":    "Gaia CLI",
+		"gaiarest":   "Gaia REST API",
+		"sdk":        "SDK",
+		"tendermint": "Tendermint",
+	}
+)
+
+func init() {
+	progName = filepath.Base(os.Args[0])
+	cwd, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	flag.StringVar(&entriesDir, "d", filepath.Join(cwd, entriesDirName), "entry files directory")
+	flag.Usage = printUsage
+}
+
+func main() {
+	log.SetFlags(0)
+	log.SetPrefix(fmt.Sprintf("%s: ", filepath.Base(progName)))
+	flag.Parse()
+
+	if flag.NArg() < 1 {
+		errInsufficientArgs()
+	}
+
+	cmd := flag.Arg(0)
+	switch cmd {
+
+	case "init":
+		initChangesDir()
```

Comment:
I think that we should delete the command `init` altogether and just check during an add if things are initialized or not during the first add.

Reply:
Init is gone